src/toast/tests/ops_noise_estim.py

```diff
@@ -110,8 +110,10 @@ def test_noise_estim(self):
         estim.apply(data)
```

Comment:
We had a unit test, why did it not catch the wrong communicators being used?

Reply:
The problems would only manifest if there were multiple groups working on multiple observations.  This may not be the case in the original data you used to develop the code.  The unit tests on github CI use 2 groups each with one process.  Running with 4 processes (2 groups of 2) made it fail more reliably.  This is a larger problem, for example the SimConviqt unit tests consistently deadlock when run on 4 processes.  We need automated testing a larger system (as you know we have been discussing that in other contexts).